<ui version="4.0" >
 <class>WidgetConnect</class>
 <widget class="QWidget" name="WidgetConnect" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Connect...</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>16</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1" colspan="2" >
    <layout class="QGridLayout" >
     <item row="0" column="1" >
      <widget class="QToolButton" name="fBtnFavourites" >
       <property name="text" >
        <string>Bookmarks</string>
       </property>
       <property name="icon" >
        <iconset resource="../application.qrc" >:/icons/icon_favourite.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="popupMode" >
        <enum>QToolButton::InstantPopup</enum>
       </property>
       <property name="toolButtonStyle" >
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="2" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Address:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy" >
        <cstring>fAddress</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2" >
      <widget class="QLineEdit" name="fAddress" >
       <property name="minimumSize" >
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Login:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy" >
        <cstring>fLogin</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2" >
      <widget class="QLineEdit" name="fLogin" >
       <property name="minimumSize" >
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="0" >
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Pasword:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy" >
        <cstring>fPassword</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2" >
      <widget class="QLineEdit" name="fPassword" >
       <property name="minimumSize" >
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
       <property name="echoMode" >
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="2" >
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>0</number>
       </property>
       <item>
        <widget class="QProgressBar" name="fProgress" >
         <property name="maximumSize" >
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="maximum" >
          <number>0</number>
         </property>
         <property name="value" >
          <number>-1</number>
         </property>
         <property name="alignment" >
          <set>Qt::AlignHCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnConnect" >
         <property name="text" >
          <string>Connect</string>
         </property>
         <property name="autoDefault" >
          <bool>true</bool>
         </property>
         <property name="default" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="3" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>21</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../application.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>fAddress</sender>
   <signal>returnPressed()</signal>
   <receiver>btnConnect</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>151</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel" >
     <x>207</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fLogin</sender>
   <signal>returnPressed()</signal>
   <receiver>btnConnect</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>128</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel" >
     <x>220</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fPassword</sender>
   <signal>returnPressed()</signal>
   <receiver>btnConnect</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>137</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel" >
     <x>208</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
